packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -51,6 +48,14 @@ import storyResponse from './db/storyResponse';
 import DocumentPane from './components/documentPane';
 import { Accessibility, Design, Priority } from './components/checklist';
 
+// Disable reason: Unable to spy on `React.createElement` when using
+// import/export. The function is added inline in the final output when
+// using import/export, but not when using require. Jasmine can then hook
+// into the require statement to swap out the function with a spy.
+// eslint-disable-next-line no-undef
```

Comment:
We could also just set `globals: { require: 'readonly' }` in the eslintrc config to avoid having to add this comment. Makes it more clear that this is a global added by webpack but still running in a browser context